deps/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ add_subdirectory(argon2)
 add_subdirectory(base64)
 add_subdirectory(irrxml)
 
+if(${CMAKE_CXX_COMPILER_ID} MATCHES GNU)
```

Comment:
This rules out clang.  Maybe change it to if it doesn't match msvc.

Reply:
Yes, because the option is gcc-specific and using just one compiler to enforce the override/final qualifiers on virtual override declarations is sufficient.